ga4gh/datamodel/__init__.py

```diff
@@ -488,3 +505,25 @@ def _getDataFilenames(self, dataDir, patterns):
 
     def getFileHandle(self, dataFile):
         return fileHandleCache.getFileHandle(dataFile, self.openFile)
+
+
+class MetadataSidecarMixin(object):
+    """
+    Loads a human readable sidecar and makes its values available as
+    self._sidecar(key). Accepts the filename from which we will load
+    the data.
+    """
+    def loadSidecar(self, filepath):
+        jsonFilename = os.path.splitext(filepath)[0] + ".json"
+        try:
+            with open(jsonFilename) as data:
+                self._sidecar = json.load(data)
+        except (ValueError, IOError):
+            raise exceptions.FileOpenFailedException(jsonFilename)
+            self._sidecar = {}
+
+    def sidecar(self, key):
```

Comment:
It's not clear what this method does, and I'm not finding the ``sidecar`` term very useful here. I would have thought ``_metadata`` would have been a more descriptive --- metadata could come from anywhere in the future, the fact we're using json files at the moment is an implementation detail.

Do we use this ``sidecar`` method anywhere? If so, I would suggest replacing with ``get_metadata(key)``, which raises a KeyError if the key is not found.